server/package.json

```diff
@@ -4,23 +4,21 @@
   "description": "This is the server app providing REST API",
   "main": "app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "./node_modules/babel-cli/bin/babel-node.js --presets node6 app.js"
+    "test": "NODE_ENV=testing nodemon --exec 'mocha --recursive -R min'",
```

Comment:
do we need specify 'test' for client side and server side, e.g., client side: npm run test = ng test

Reply:
This script is only used in /server sub folder. We could add server side test script in root folder in the future.